docker/README.md

```diff
@@ -0,0 +1,48 @@
+# Docker deployment
```

Comment:
Even though this is a nice README it's hidden in this directory, could be of great help for users if documented in Edge docs instead of here https://wiki.polygon.technology/docs/edge/get-started/set-up-ibft-locally, wdyt?

Reply:
Great idea, and the users would definitely benefit from having something similar to this `README` in the public docs.    
We should probably do that, once `feature/v3-parity` is merged into develop and we have a release including this feature.    

I don't think that we have any information regarding `polybft` in our public docs yet.